common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +456,199 @@ def test_teams_membership(self):
             self.browse_teams_page.team_cards[0].find_element_by_css_selector('.member-count').text,
             '1 / 10 Members'
         )
+
+    def test_navigation_links_are_working_properly(self):
+        """
+        Scenario: User should be able to navigate to "browse all teams" and "search team description" links.
+        Given I am enrolled in a course with a team configuration and a topic
+            containing one team
+        When I visit the Teams page for that topic
+        Then I should see the correct page header
+        And I should see the link to "browse all team"
+        And I should navigate to that link
+        And I see the relevant page loaded
+        And I should see the link to "search teams"
+        And I should navigate to that link
+        And I see the relevant page loaded
+        """
+        self.browse_teams_page.visit()
+        self.verify_page_header()
+
+        self.assertTrue(self.browse_teams_page.browse_all_teams_link_present())
+        self.browse_teams_page.click_browse_all_teams_link()
+        self.topics_page.is_browser_on_page()
+
+        self.browse_teams_page.visit()
+        self.verify_page_header()
+        self.assertTrue(self.browse_teams_page.search_team_link_present())
+        self.browse_teams_page.click_search_team_link()
+        # TODO Add search page expectation once that implemented.
+
+
+@attr('shard_5')
+class CreateTeamTest(TeamsTabBase):
+    """
+    Tests for creating a new Team within a Topic on the Teams page.
+    """
+
+    def setUp(self):
+        super(CreateTeamTest, self).setUp()
+        self.topic = {u"name": u"Example Topic", u"id": "example_topic", u"description": "Description"}
+        self.set_team_configuration({'course_id': self.course_id, 'max_team_size': 10, 'topics': [self.topic]})
+        self.browse_teams_page = BrowseTeamsPage(self.browser, self.course_id, self.topic)
+        self.browse_teams_page.visit()
+        self.browse_teams_page.is_browser_on_page()
```

Comment:
I don't think this line provides any value. Are you expecting it to raise an exception?

source: https://bok-choy.readthedocs.org/en/latest/api_reference.html?highlight=is_browser_on_page#bok_choy.page_object.PageObject.is_browser_on_page